modules/rosa-hcp-sharing-vpc-dns-and-roles.adoc

```diff
@@ -0,0 +1,105 @@
++
+[source,terminal]
+----
+$ rosa create oidc-config
+----
++
+You receive confirmation that the command created an OIDC configuration:
++
+[source,terminal]
+----
+I: To create Operator Roles for this OIDC Configuration, run the following command and remember to replace <user-defined> with a prefix of your choice:
+	rosa create operator-roles --prefix <user-defined> --oidc-config-id 25tu67hq45rto1am3slpf5lq6jargg
+----
+
+. Create the account roles by entering the following command:
++
+[source,terminal]
+----
+$ rosa create account-roles --oidc-config-id <oidc-config-ID> <1>
+    --installer-role-arn <Installer_Role> <2>
+    --route53-role-arn <Created_Route_53_Role_Arn> <3>
+    --vpc-endpoint-role-arn <Created_VPC_Endpoint_Role_Arn> <4>
+    --prefix <operator-prefix> <5>
+----
++
+--
+<1> Provide the OIDC configuration ID that you created in the previous step.
+<2> Provide your installer ARN that was created as part of the `rosa create account-roles` process.
+<3> Provide the ARN for the Route 53 role that the *VPC Owner* created.
+<4> Provide the ARN for the VPC endpoint role that the *VPC Owner* created.
+<5> Provide a prefix for the Operator roles.
+--
+
+. Create the Operator roles by entering the following command:
++
+[source,terminal]
+----
+$ rosa create operator-roles --oidc-config-id <oidc-config-ID> <1>
+    --installer-role-arn <Installer_Role> <2>
+    --route53-role-arn <Created_Route_53_Role_Arn> <3>
+    --vpc-endpoint-role-arn <Created_VPC_Endpoint_Role_Arn> <4>
+    --prefix <operator-prefix> <5>
+----
++
+--
+<1> Provide the OIDC configuration ID that you created in the previous step.
+<2> Provide your installer ARN that was created as part of the `rosa create account-roles` process.
+<3> Provide the ARN for the Route 53 role that the *VPC Owner* created.
+<4> Provide the ARN for the VPC endpoint role that the *VPC Owner* created.
+<5> Provide a prefix for the Operator roles.
+--
++
+[NOTE]
+====
+The Installer account role and the shared VPC roles must have a one-to-one relationship. If you want to create multiple shared VPC roles, you should create one set of account roles per shared VPC role.
+====
+
+. After you create the Operator roles, share the full domain name, which is created with `<intended_cluster_name>.<reserved_dns_domain>`, your _Ingress Operator Cloud Credentials_ role's ARN, your _Installer_ role's ARN, and your _Control plane Operator Cloud Credentials_ role's ARN with the *VPC Owner* to continue configuration.
++
+The shared information resembles these examples:
++
+* ``my-rosa-cluster.14eo.p1.openshiftapps.com``
+* ``arn:aws:iam::111122223333:role/ManagedOpenShift-Installer-Role``
+* ``arn:aws:iam::111122223333:role/my-rosa-cluster-openshift-ingress-operator-cloud-credentials``
+* ``arn:aws:iam::962993289388:role/route53-scoped"
```

Comment:
This comes from the vpc owner so I don't think it makes sense to have to include this here as the vpc owner would already be aware of these